_projects/_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
+# add a description of your project
+requirements:
+ - The proposal has to contain a clear concept.
+ - patch on any astropy affiliated package.
+difficulty: low
+issues:
+ - https://github.com/astropy/astropy/issue/4444
+ - https://github.com/astropy/astropy/issue/5555
+mentors:
+ - astrofrog
+initiatives:
+ - GSOC
+ - SOCIS
+tags:
+ - python
+collaborating_projects:
+ - astropy
```

Comment:
Again, why is this here?  Isn't the file going to already be *in* the astropy section?